code/modules/atmospherics/machinery/components/binary_devices/pump.dm

```diff
@@ -61,6 +75,25 @@
 	if(air1.pump_gas_to(air2, target_pressure))
 		update_parents()
 
+/obj/machinery/atmospherics/components/binary/pump/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/assembly/motor))
+		if(motor_numbers == max_motors)
+			to_chat(user, "<span class='warning'>There are already [max_motors] connected to the pump!</span>")
+			return TRUE
+		motor_numbers++
+		update_assembly()
+		qdel(W)
+		playsound(get_turf(src), 'sound/items/handling/component_pickup.ogg', 35, TRUE)
```

Comment:
You already defined the pickup sound on the component.
```
var/obj/item/assembly/motor/motor = W
playsound(get_turf(src), motor.pickup_sound, 35, TRUE)
```

Reply:
that's the sound of when you use the component on the device, without it does it play the sound regardless?

Reply:
you still have to play the sound, but no need to define the sound again if it is just going to play pickup sound.